awebox/pmpc.py

```diff
@@ -112,6 +113,15 @@ def __build_trial(self):
         self.__trial.nlp.build(self.__trial.options['nlp'], self.__trial.model, self.__trial.formulation)
         self.__trial.visualization.build(self.__trial.model, self.__trial.nlp, 'MPC control', self.__trial.options)
 
+        # remove state constraints at k = 0
```

Comment:
For k=0, the state is constrained anyways by the initial condition, right?

Reply:
Indeed. For k=0, one should in practice always remove the constraints that solely depend on the states (nonlinear constraints or simple bounds), because it can always happen (through disturbances, or numerical inaccuracies when operating at an active constraint) that your initial state, which is fixed, violates these constraints, rendering the problem infeasible.